packages/@okta/vuepress-site/docs/guides/authenticators-overview/main/index.md

```diff
@@ -1,8 +1,36 @@
 ---
-title: Authenticators overview
+title: Authenticators Overview
 ---
 
 <ApiLifecycle access="ie" /><br>
 <ApiLifecycle access="Limited GA" /><br>
 
-As you build your authentication use cases in your app, Okta gives you the option to further protect user sign-ins through the use of authenticators.
+Increase the security of your app by requiring a user to verify their identity in more than one way.
+
+---
+
+## Overview
+
+Strengthen your application's sign-in process by adding multiple **authentication factors**, or ways for a user to confirm their identity. Factors are enabled in the Okta org by creating a **policy** with one or more authenticators, and then assigning that policy to your app. A policy manages both the types of authenticators and which ones are required or optional. **Multifactor Authentication** (MFA) means that the policy requires at least two factors for a user to sign in. For example, an application can require using Okta Verify in addition to a username and password.
```

Comment:
The sentence "Factors are enabled" is a little confusing to me ... just the use of factors, really.  Same with the first "Authentication factors" .... I think these are better termed "Authenticators".  All Authenticators have factor types, which is why the user of just factors might be confusing. See: https://help.okta.com/oie/en-us/Content/Topics/identity-engine/authenticators/configure-authenticators.htm

Just my thoughts on this. Best to get a SME to chime in.

Reply:
The wording was pre-approved, but I've added the link to that page and its parent page listing all the authenticators.